docsbox/Dockerfile

```diff
@@ -3,7 +3,10 @@ MAINTAINER Dmitry Veselov <d.a.veselov@yandex.ru>
 
 RUN apt-get update && apt-get upgrade -y
 RUN apt-get install -y libffi-dev libmagic-dev libmagickwand-dev
-RUN apt-get install -y libreoffice libreofficekit-dev
+RUN apt-get install software-properties-common -y
+RUN add-apt-repository ppa:libreoffice/ppa
+RUN apt update
```

Comment:
It would be better to use apt-get or apt, not both. Also, apt-get update is ran already above, if you add the repository before that, we dont need to run it again.

Reply:
I changed the apt command for apt-get
The order I had to keep:
- software-properties-common needs to run after apt-get update 
- add-apt-repository needs software-properties-common to run
- and after add-apt-repository I need to run apt-get update again